Xamarin.Forms.Build.Tasks/TypeReferenceExtensions.cs

```diff
@@ -324,6 +324,38 @@ public static MethodReference GetImplicitOperatorTo(this TypeReference fromType,
 			return null;
 		}
 
+		public static TypeReference ResolveGenericParameters(this TypeReference self, MethodReference declaringMethodReference)
```

Comment:
don't we have this already ?

Reply:
What method do you think can be alternative for this?